microsoft-edge/devtools-guide-chromium/whats-new/2022/07/devtools-104.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What's new in DevTools (Microsoft Edge 104)
+description: "Use Layers in 3D View. Improved display of request URLs in Issues tool. Improving accessibility for the Detached Elements tool. Fix: Arrows representing web socket traffic now render correctly in the Network tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 07/21/2022
+---
+# What's New in DevTools (Microsoft Edge 104)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Use Layers in 3D View
+<!-- rank: 1 -->
+
+<!-- todo: check work item -->
+
+<!-- Title: Where did Layers go? -->
+<!-- Subtitle: Layers isn't a stand-alone tool anymore, it's wrapped up in the comprehensive 3D View; a tool for all your visual debugging needs. -->
+
+The standalone **Layers** tool has now been completely removed.  During the phase-out period, the **Layers** tool remained, as an empty panel containing a link that opened the **3D View** tool.
+
+In the **3D View** tool, the **Composited Layers** view has all the features you love from the **Layers** tool, along with some updates.  The **Composited Layers** view now sits alongside even more visual debugging tools.  Try out the **DOM** and **Z-index** views for debugging while you're in the **3D View** tool, and you'll see how these three views work together.
```

Comment:
The **Composited Layers** view is complemented by two other visual debugging panes.